content/api/official/python/examples.md

```diff
@@ -22,6 +22,24 @@ Create the topic used to produce and consume records:
 fluvio topic create python-data
 ```
 
+### Login
+
+Login to Infinyon Cloud using username/password
+
+%copy%
+```shell
+from fluvio import cloud
+cloud.login(email="my@email.com", password="mypassword")
+```
+
+Alternatively, you can login using `--ouath2`. However, this method is only suitable for interactive mode, where you can copy/paste the link to a browser for authorization.
```

Comment:
```suggestion
Alternatively, you can login using `--oauth2`. However, this method is only suitable for interactive mode, where you can copy/paste the link to a browser for authorization.
```

there is a typo there. Also not sure if `--` is needed here since we are not in CLI context